### ISSUE TITLE
Error logging in to server.

### ISSUE THREAD

**Issue by oleg-koval** [ati/ljsm#17]
After running:

```shell
perl ljsm.pl -a -I -i 3 -u user:password user
```

Received:

```
logging in to http://www.livejournal.com/...
Error logging in to server.%
```

`user:password` combination triple-checked :)

macOs Catalina 10.15.2

**Comment by dimkin-eu** [comment]
@ati такая же ошибка
Catalina 10.15.7
'''
Error logging in to server.zzz:lj xxx$ perl ljsm.pl -I -i 3 -u user:pass user
logging in to http://www.livejournal.com/... 
Error logging in to server.zzz:lj xxx$ 
'''

**Comment by n1313** [comment]
@oleg-koval @dimkin-eu если у вас в пароле много спец-символов (например, `&`), то он неправильно воспринимается скриптом. Попробуйте обернуть пару логин:пароль в одинарные кавычки, типа `perl ljsm.pl -I -i 3 -u 'kukutz:JKHkjhnbsh66mhFTbhf' kukutz`

**Comment by dimkin-eu** [comment]
@n1313 сделал пароль просто alphanumeric - не проканало в обоих случаях
```
 lj-back % perl ljsm.pl -I -i 3 -u 'dimkin-lv:zQXV4RbhBJkv1234' dimkin-lv
logging in to http://www.livejournal.com/... 
Error logging in to server.
```

```
% perl ljsm.pl -I -i 3 -u dimkin-lv:zQXV4RbhBJkv1234 dimkin-lv 
logging in to http://www.livejournal.com/... 
Error logging in to server.% 
```

**Comment by n1313** [comment]
Да, я вот после исправления проблемы со спец-символами в пароле столкнулся с этой же самой проблемой. Думал, меня за предыдущие неправильные попытки забанили...

**Comment by dimkin-eu** [comment]
@n1313 тоже думал об этом, но с тем же паролем с того же хоста пускают

**Comment by danil-smirnov** [comment]
Да, на линуксе не работает

**Comment by trankov** [comment]
`https://github.com/ati/ljsm/pull/16/files`

По этому адресу есть pull request с исправленной ошибкой, добавьте выделенные строки в perl-файл.
Затем установите нужную библиотеку: `sudo cpan Mozilla:CA`
Потом запускайте скрипт, как в инструкции.

Стандартный коннектор perl не дружит с новыми SSL-требованиями от LiveJournal. Данный фикс помогает решить эту проблему.

**Comment by danil-smirnov** [comment]
@trankov Всё сделал но получаю ту же самую `Error logging in to server.`

**Comment by danil-smirnov** [comment]
Мне удалось запустить софтину после того как я:
1. Поменял [протокол BASE_URL](https://github.com/ati/ljsm/blob/master/ljsm.pl#L74) с `http` на `https`
2. Установил `LWP::Protocol::https`
3. Добавил `use LWP::Protocol::https;` в код
@trankov Будет круто если добавишь в свой PR ;)

**Comment by trankov** [comment]
>    @trankov Будет круто если добавишь в свой PR ;)

Это вообще ни разу не мой PR, я просто пошарил по репозиторию и обнаружил чей-то ждущий аппрува коммит. Внёс исправления ручками и погуглил как на Perl ставятся пакеты.

**Comment by trankov** [comment]
> @trankov Всё сделал но получаю ту же самую `Error logging in to server.`

Судя по комментам у вас LInux, у меня MacOS. По идее одно и то же, но, видимо, какие-то нюансы в системном окружении сыграли свою роль. Неохота разбираться, какие именно ))
